plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/GenericCatalog.java

```diff
@@ -149,7 +150,8 @@ public DBSObject getChild(@NotNull DBRProgressMonitor monitor, @NotNull String c
     public Class<? extends DBSObject> getPrimaryChildType(@Nullable DBRProgressMonitor monitor)
         throws DBException
     {
-        if (!CommonUtils.isEmpty(schemas) || (monitor != null && !CommonUtils.isEmpty(getSchemas(monitor)))) {
+        if (!CommonUtils.isEmpty(schemas) || !CommonUtils.isEmpty(getSchemas(monitor == null ?
```

Comment:
Not sure about it. Triggering schema load for property test is usually a bad idea (hint: schema load can take a lot of time in some cases).